frontend/src/components/MapView/DateSelector/TimelineItems/index.tsx

```diff
@@ -25,6 +25,18 @@ type DateItemStyle = {
   emphasis?: string;
 };
 
+export const LIGHT_BLUE_HEX = '#C0E8FF';
+const DARK_BLUE_HEX = '#00A3FF';
+
+export const LIGHT_GREEN_HEX = '#B5F8BB';
+const DARK_GREEN_HEX = '#68CC71';
+
+const LIGHT_ORANGE_HEX = '#F9CEC1';
+const DARK_ORANGE_HEX = '#FF9473';
+
+// const LIGHT_YELLOW_HEX = '#FFF176';
+// const DARK_YELLOW_HEX = '#FEC600';
```

Comment:
Are these needed?

Reply:
I wanted to keep them around in case we want to use a 4th color, but not, not really needed, will remove